civihr_default_theme/scss/pages/_users-list.scss

```diff
@@ -19,19 +19,19 @@
   #edit-actions {
     margin-bottom: 0;
 
-    #edit-actions-submit {
-      @include button-variant($btn-primary-color, $btn-primary-bg, $btn-primary-border);
-
-      margin-right: 20px;
-    }
-
     > a {
       background: #767b8e;
       border: 0;
-      color: #fff;
+      color: $light;
       display: inline-block;
       margin-left: 20px;
       padding: 0.7rem 1.1rem;
     }
   }
+
+  #edit-actions-submit {
```

Comment:
why is this new style needed? No mention in the PR description

Reply:
This is not new, its moved to avoid more than 3 nested selectors.